script/create-dist

```diff
@@ -59,6 +59,12 @@ BINARIES = {
     'libdevtools_discovery.a',
     'libdevtools_http_handler.a',
     'libhttp_server.a',
+    'libdesktop_capture.a',
```

Comment:
Can you make the lists in `darwin` and `linux` have the same order? And should `libdesktop_capture_differ_sse2` also be listed here?